### PR TITLE
feat: all of sprint 1

### PR DESCRIPTION
**Pull request by JohaneDecamps**

### CODE REVIEW

**Review by Pierre-Voisin — APPROVED** [review]

**Review by aurelienLavanchy — APPROVED** [review]

**Review by RiZainou — APPROVED** [review]

**Review by RiZainou — APPROVED** [review]

File: client/src/App.css [hunk 1]
```diff
@@ -1,165 +1,68 @@
...
-  background-image: url("./assets/images/background.webp");
-  background-size: cover;
-  background-position: top;
-  position: relative;
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-  padding: 30px;
-}
+@import url("https://fonts.googleapis.com/css2?family=Indie+Flower&display=swap");
```
Comment (ioayoub) [inline client/src/App.css:1]: prefer downloading fonts here

File: client/src/App.css [hunk 2]
```diff
@@ -1,165 +1,68 @@
...
-
-.wcs:hover {
-  border-bottom: solid 1px #ff1493ad;
+.mainButton {
+  margin-top: 2rem;
+  margin-bottom: 3rem;
+  width: 6.3rem;
+  height: 2.5rem;
+  background-color: #ff8c6f;
+  /* border : 2px inset #3d3d3d; */
```
Comment (ioayoub) [inline client/src/App.css:46]: please remove this unused line

File: client/src/App.jsx [hunk 3]
```diff
@@ -1,90 +1,36 @@
...
+  const [imageURL, setImageURL] = useState([]);
+  useEffect(() => {
+    fetch("https://www.themealdb.com/api/json/v1/1/filter.php?a=Canadian")
+    .then((response) => response.json())
+    .then((data) => {
+      const tempData = data.meals.slice(0, 4);
+      setImageURL(tempData.map((m) => m.strMealThumb))
+    })
+  }, []);
+  // console.info(imageURL)
```
Comment (ioayoub) [inline client/src/App.jsx:17]: 👀

File: client/src/App.jsx [hunk 4]
```diff
@@ -1,90 +1,36 @@
...
+  return (
+    <>
+      <Navbar />
+      <div className="container">
+        <input type="text" placeholder="Enter your name" className="inputName" />
+        <button type="button" className="mainButton"> Start </button>
+        <h2> Most Popular </h2>
+            <div className="featuredImages">
+              {imageURL && imageURL.map((i, index) => {
+                const id = index + 0.1
```
Comment (ioayoub) [inline client/src/App.jsx:28]: huh ?

**Review by ioayoub — CHANGES_REQUESTED** [review]

File: client/src/App.css (hunk 2 above)
Comment (aurelienLavanchy) [reply client/src/App.css]: :heavy_check_mark:

File: client/src/App.jsx (hunk 3 above)
Comment (aurelienLavanchy) [reply client/src/App.jsx]: :facepalm:

File: client/src/App.jsx (hunk 4 above)
Comment (aurelienLavanchy) [reply client/src/App.jsx]: fixed

**Review by ioayoub — APPROVED** [review]

**Review by RiZainou — APPROVED** [review]

**Pull request merged**